js/cli/src/item.ts

```diff
@@ -73,6 +73,7 @@ programCommand("create_item_class")
         parentMint: config.parent
           ? new web3.PublicKey(config.parent.mint)
           : null,
+        parentOfParentClass: null,
```

Comment:
This is not correct - I'm not sure what it should be though.

Reply:
```suggestion
        parentOfParentClass: config.parent?.parent
           ? new web3.PublicKey(config.parent.parent.key)
           : null,
```

I don't see any other conventions used in the code that make me stuck on using `key` or `id` for the property name. So, dealers choice here. 😃